pytorch_lightning/trainer/trainer.py

```diff
@@ -196,6 +197,8 @@ def __init__(
 
             gradient_clip_val: 0 means don't clip.
 
+            gradient_clip_algorithm: 'value' means clip_by_value, 'norm' means clip_by_norm. Defualt: 'norm'
```

Comment:
```suggestion
            gradient_clip_algorithm: 'value' means clip_by_value, 'norm' means clip_by_norm. Default: 'norm'
```

Reply:
All modifications have been completed based on comments. Can you check the last changes? 
https://github.com/PyTorchLightning/pytorch-lightning/pull/6123/commits/e92ec69bc7af1d6c2694141617bed9e07d4d1a67
Thanks.